server/graphql/v2/object/TransactionsImport.ts

```diff
@@ -86,15 +89,54 @@ export const GraphQLTransactionsImport = new GraphQLObjectType({
     rows: {
       type: new GraphQLNonNull(GraphQLTransactionsImportRowCollection),
       description: 'List of rows in the import',
-      resolve: async importInstance => {
-        const where = { TransactionsImportId: importInstance.id };
+      args: {
+        ...getCollectionArgs({ limit: 100 }),
+        status: {
+          type: GraphQLTransactionsImportRowStatus,
+          description: 'Filter rows by status',
+        },
+        searchTerm: {
+          type: GraphQLString,
+          description: 'Search by text',
+        },
+      },
+      resolve: async (
+        importInstance,
+        args: { limit: number; offset: number; status: TransactionsImportRowStatus; searchTerm: string },
+      ) => {
+        const where: Parameters<typeof TransactionsImportRow.findAll>[0]['where'] = {
+          [Op.and]: [{ TransactionsImportId: importInstance.id }],
+        };
+
+        // Filter by status
+        if (args.status) {
+          if (args.status === 'IGNORED') {
```

Comment:
🍨 Use the enum.

Reply:
I don't have a strong opinion on that one. `args.status` is typed, so if you try to do something like `args.status === 'xxx'` typescript will error, because it knows the possible values for it. Knowing this, what's the added value of using the enum?